<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfFilterPlot</class>
 <widget class="QDialog" name="WkfFilterPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter Visualization</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="UtQtGL2DPlotWidget" name="plotWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>4</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="xAxisLabel">
         <property name="text">
          <string>X-axis Column</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="xAxisComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="yAxisLabel">
         <property name="text">
          <string>Y-axis Column</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="yAxisComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="seriesLabel">
         <property name="text">
          <string>Series Column</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="seriesComboBox"/>
       </item>
       <item>
        <widget class="QCheckBox" name="seriesCheckBox">
         <property name="text">
          <string>Enable Series</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="checkBoxList">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0,0,0,0)</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionRectVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtGL2DPlotWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>UtQtGL2DPlotWidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
